security-proxy/security-proxy.properties

```diff
@@ -20,13 +20,12 @@ userSearchBaseDN=ou=users
 # the second part of looking up the user
 userSearchFilter=(uid={0})
 # The base DN to use for looking up the roles/groups/authorities of the logged in user.  Normally the ldap is configured like:
-#   ou=groups
-#     ou=somegroup
+#   ou=roles
+#     ou=somerole
```

Comment:
key for role/group is `cn`, so I think, here, you can change ou=somerole to `cn=somerole`.